Source/Engine/Level/SceneObjectsFactory.cpp

```diff
@@ -238,6 +239,8 @@ void SceneObjectsFactory::HandleObjectDeserializationError(const ISerializable::
     {
         const Guid parentId = JsonTools::GetGuid(parentIdMember->value);
         Actor* parent = Scripting::FindObject<Actor>(parentId);
+        DummyScript* dummyScript = parent->AddScript<DummyScript>();
```

Comment:
Check if `parent ` is valid as it might be `null`.

Reply:
Moved everything couple lines down into already existing `if(parent)`